cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
Is this file needed or just an artefact from Cypress install? Delete it if not?

Reply:
It's an artifact from the Cypress install. I left it in so the general file structure was there but if we'd rather not have it for now I can remove it

Reply:
That's all good. We can leave it for future tests then. Thanks!